### ISSUE TITLE
Unable to change effect pass material parameters

### ISSUE THREAD

**Issue by taphos** [vanruesc/postprocessing#152]
In version 6.1.1 it was possible to to make effect pass material transparent, but not in version 6.8.1

I have render pass and custom outline effect pass both rendered directly to screen, to take advantage of hardware antialiasing (instead of using SMAAEffect). To be able to do that I have to make effect pass material transparent.

```
// Render directly to screen to use MSAA
renderPass.renderToScreen = true;
composer.addPass(renderPass);

const outlinePass = new EffectPass(camera, customOutlineEffect);
outlinePass.renderToScreen = true;
// Blit material transparent because RenderPass is rendered to screen
outlinePass.getFullscreenMaterial().transparent = true;
composer.addPass(outlinePass);
```

since version 6.8.1 `outlinePass.getFullscreenMaterial()` returns null

**Comment by vanruesc** [comment]
Thanks for the bug report!

The `EffectMaterial` creation was moved into the `Pass` initialization phase in 0c59930 to make the effect initialization more flexible.

You can fix the issue in your code by calling `getFullscreenMaterial` after adding the pass:
```js
composer.addPass(outlinePass);
outlinePass.getFullscreenMaterial().transparent = true;
```

This is just a temporary solution, though. I'll add a fix later to make this more robust.

**Comment by vanruesc** [comment]
Fixed in `postprocessing@6.8.2`.
